_docs_integrate/delete-attributes.md

```diff
@@ -32,10 +32,12 @@ Doing so, also a LocalAttribute with the same `content` and a respective `shareI
 In the case of RelationshipAttributes we have own shared and peer shared RelationshipAttributes analogeously, however, no unshared LocalAttributes like we have in the case of IdentityAttributes with RepositoryAttributes.
 These different kinds of Attributes have different demands that need to be taken into account, wanting to delete them.
 
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/04c48548-90a3-4e9e-839a-4bb9a5dddcfb" id="9hZD~iRvE8aL"></iframe></div>
```

Comment:
I really like the diagram of the flow overall. I think it should be slightly modified for the case of "own shared Attributes", as the path of deletion of those is not so easy to follow because the term own shared Attribute appears (for understandable reasons) very late. It is also possible to run through the diagram without deleting an Attribute (if the Integrator initially wanted to delete an own shared Attribute and then decides to keep the history (which corresponds to the own shared Attribute) anyway).

However, it is not so easy to redirect the paths accordingly and possibly introduce a second "Call DeleteOwnSharedAttributeAndNotifyPeer use case" box without becoming too redundant or having to make other compromises. So perhaps you could bring clarity to the diagram by adding boxes of an additional color as a conclusion as soon as the type of Attribute you want to delete is clear. For example, between the box "Do you want to delete a RepositoryAttribute?" and the box "Do you want the peer to delete their copy of the Attribute, too?", that are connected via a "No", a box with the name "-> The Attribute you want to delete is therefore an own shared Attribute." (that links to the LocalAttribute in the Data Model Overview) could be added.

As I said, I'm not sure how exactly or if you can adjust the path of deleting the own shared Attributes appropriately, but wanted to share with you that I had trouble understanding it.